.maestro/testsV2/subFolder/cloud/commands/TapOnBack.yml

```
@@ -0,0 +1,3 @@
+appId: ${MAESTRO_APP_ID}
+---
+- pressKey: back
```

voir message précédent ⬆️ 

et c'est le flow qui détermine comment revenir en arrière

```suggestion
- runFlow: # Permet de retourner sur la page précédente (cloud)
    when:
      true: ${MAESTRO_RUN_CLOUD_COMMANDS}
    commands:
    	- pressKey: back

- tapOn: # Permet de retourner sur la page précédente (local)
    id: 'com.android.systemui:id/back'
    optional: true
```

ça permettra
* d'éviter la duplication
* de changer à un seul endroit si demain on change la manière de revenir en arrière sur le cloud ou le local
* d'éviter d'oublier de faire un back en local et pas en cloud (ou vice versa)

---

Ca arrive dans mon prochain commit, effectivement c'est plus propre de cette façon, je retiens pour la suite !